### DISCUSSION TITLE
Polyhaven Textures

### DISCUSSION THREAD

**Discussion by JJL772** [StrataSource/Portal-2-Community-Edition#1756] [Resources]
## Description

Download the VTFs and VMTs here: https://github.com/JJL772/polyhaven-source/releases/latest/download/polyhaven-textures.zip

This is a pack of PBR textures from https://polyhaven.com. Some of the materials may need tweaks (i.e. to add $surfaceprop).

These were batch imported using [this script](https://github.com/StrataSource/sdk_tools/blob/main/scripts/get-polyhaven-textures.py).

## Screenshots

![Screenshot_20241124_012202](https://github.com/user-attachments/assets/d0485822-7c00-440a-8242-e744542a1a24)
![Screenshot_20241124_012227](https://github.com/user-attachments/assets/5161ad34-9bc6-45c6-8bb0-35ff11e3b7c5)
![Screenshot_20241124_012256](https://github.com/user-attachments/assets/eea2cd88-299b-4e0c-8683-4bf128deac55)
![Screenshot_20241124_012955](https://github.com/user-attachments/assets/1badc0c1-dd21-4a48-9a1d-15f08a3b44a4)
![Screenshot_20241124_012354](https://github.com/user-attachments/assets/f43068bf-b831-40d6-92fe-f7c7ff423967)

## Credits

Textures are from https://polyhaven.com and are [licensed under CC0](https://polyhaven.com/license). Credit is not _required_, but you should still credit them!

The VMTs can be considered public domain too.

## Download

https://github.com/JJL772/polyhaven-source/releases/latest/download/polyhaven-textures.zip

**Comment by TwoKrazy** [comment]
yo that's actually sick!
